docs/openshift.asciidoc

```diff
@@ -96,10 +103,10 @@ Then we are going to register the Debezium MySQL connector to run against the de
 
 [source%nowrap,bash]
 ----
-oc exec -i kafka-0 -- curl -X POST \
+oc exec -i my-cluster-kafka-0 -- curl -X POST \
     -H "Accept:application/json" \
     -H "Content-Type:application/json" \
-    http://kafka-connect:8083/connectors -d @- <<'EOF'
+    http://my-connect-cluster-connect:8083/connectors -d @- <<'EOF'
```

Comment:
What did make you change the name?

Reply:
The default name is changed in the new template

Reply:
Hum, but "my-connect-cluster-connect" really seems odd. Can we make it "my-connect-cluster" at least (ignoring the fact that it isn't really a cluster but just a single node).